HW1/P5/P5.txt

```diff
@@ -1 +1,13 @@
-# Your discussion here
+From Harvard University to Kevin Bacon: each path takes 3 steps
+
+	[u'Harvard_University', u'Gilmore_Girls', u'Liza_Weil', u'Kevin_Bacon']
+	[u'Harvard_University', u'Jack_Lemmon', u'Rod_Steiger', u'Kevin_Bacon']
+	[u'Harvard_University', u'Cornell_University', u'Edmund_Bacon', u'Kevin_Bacon']
+
+From Kevin Bacon to Harvard University: each path takes 2 steps
+
+	[u'Kevin_Bacon', u'Time_(magazine)', u'Harvard_University']
+	[u'Kevin_Bacon', u'Marisa_Silver', u'Harvard_University']
+
+
+
```

Comment:
Hmm..I got different results here. I had 120 paths between Harvard -> Kevin Bacon and my shortest path from Kevin Bacon -> Harvard was 5 steps. I may be wrong, would be interesting to compare..